### PR TITLE
docs: Update README.md for public-dns-with-cloud-dns example

### PR DESCRIPTION
**Pull request by danielpanzella**
Added required and renamed values to the example tfvars

- `allowed_inbound_cidrs` is a required variable
- `domain` seems to have renamed to `domain_name`

### CODE REVIEW

File: examples/public-dns-with-cloud-dns/README.md [hunk 1]
```diff
@@ -79,24 +79,26 @@ Note: The domain and subdomain association might take a few minutes to reflect.
...
-  subdomain  = "venky"
-  domain     = "wandb.ml"
+  project_id            = "playground-111"
+  region                = "us-west4"
+  zone                  = "us-west4-a"
+  namespace             = "venky-unique-3"
+  license               = "eyJhbGciOiJS6InUzhEUXM1M0xQY09yNnZhaTdoSlduYnF1bTRZTlZWd1VwSWM9In0.eyJjb25jdXJyZW50QWdlbnRzIjoxMCwiZGVwbG95bWVudElkIjoiNGU0YWNiZmYtY2E5NS00MmRiLThmYmItMjliNmY5NTI2OWE0IiwibWF4VXNlcnMiOjQsIm1heFN0b3JhZ2VHYiI6MTAwMDAwMCwibWF4VGVhbXMiOjEsImV4cGlyZXNBdCI6IjIwMjItMTAtMjBUMTY6MjY6NTUuNzA3WiIsImZsYWdzIjpbIlNDQUxBQkxFIiwibXlzcWwiLCJzMyIsInJlZGlzIiwiTk9USUZJQ0FUSU9OUyIsInNsYWNrIiwibm90aWZpY2F0aW9ucyIsIk1BTkFHRU1FTlQiLCJvcmdfZGFzaCIsImF1dGgwIl0sInRyaWFsIjpmYWxzZSwiYWNjZXNzS2V5IjoiNzk3M2FkOWItNThmOC00OTUxLWJhOTctOGQ2NGFkYzI1ZThlIiwic2VhdHMiOjQsInRlYW1zIjoxLCJzdG9yYWdlR2lncyI6MTAwMDAwMCwiZXhwIjoxNjY2MjgzMjE1fQ.O_6D3Av9QoWI16ybg54KFvs7eGWugSXPxmfhobtZe3TBFvd8PwmSCAmMojmKWsqg6KNjLJ9sjxOP_3Pj9OAdrkx5WzU0KTcIByXD2hS9VwyYUOYEohBn65oCLnQJLYphXJBrB9JVS0GSUGxR1AzwnUK1PuKZ6jQFrpt-feQOD3rvCdyM1eBQ73rdHk6zfEBmdiZ7C4LiRLV8OEMxUfwxASvVF_cFUEeVQx82AaxRwfPBLZxXTL4qlQOIFjAKwGVyDMEWq04BhQ_ASdyND45w5qXiUOlvFOergrFyGBSHg-9yDT4fhdkDw5puGthDaMFsn02rr0eYHuxKFWSY958aig"
```
Comment (jsbroks) [inline examples/public-dns-with-cloud-dns/README.md:98]: we shhoudn't be exposing a license

File: examples/public-dns-with-cloud-dns/README.md [hunk 2]
```diff
@@ -79,24 +79,26 @@ Note: The domain and subdomain association might take a few minutes to reflect.
...
+  subdomain             = "venky"
+  domain_name           = "wandb.ml"
+  allowed_inbound_cidrs = ["97.120.89.6/32"]
```
Comment (jsbroks) [inline examples/public-dns-with-cloud-dns/README.md:101]: nice ip... should I ddos?

**Review by jsbroks — CHANGES_REQUESTED** [review]

File: examples/public-dns-with-cloud-dns/README.md (hunk 1 above)
Comment (danielpanzella) [reply examples/public-dns-with-cloud-dns/README.md:98]: I didn't add that, just reformatted, but I can take it out if desired

File: examples/public-dns-with-cloud-dns/README.md (hunk 2 above)
Comment (danielpanzella) [reply examples/public-dns-with-cloud-dns/README.md:101]: is that my IP, who knows...I can change the recommended to 0.0.0.0/0 if you want

Comment (danielpanzella) [reply examples/public-dns-with-cloud-dns/README.md:101]: I just didn't want to put a not real internet facing ip address and confuse someone in the future

**Review by jsbroks — APPROVED** [review]

**Comment by jsbroks** [comment]
This PR is included in version 3.0.5 :tada:

**Pull request merged**